packages/code-du-travail-frontend/pages/themes/[slug].js

```diff
@@ -49,20 +49,31 @@ const getBreadcrumbs = (items = []) => {
 
 // Theme page
 class Theme extends React.Component {
-  static async getInitialProps({ query: { slug } }) {
-    const response = await fetch(`${API_URL}/themes${slug ? `/${slug}` : ""}`);
-    if (!response.ok) {
-      return { statusCode: response.status };
+  static async getInitialProps({ query: { slug: query } }) {
+    const [searchThemeResponse, searchResults] = await Promise.all([
+      fetch(`${API_URL}/themes${query ? `/${query}` : ""}`),
+      fetchSearchResults({
+        query,
+        excludeSources: "themes"
+      })
+    ]);
+    if (!searchThemeResponse.ok) {
```

Comment:
you should tests both `searchResults` and `searchThemeResponse` ?

Reply:
searchThemeResponse handle things differently from the search api which throws